<launch>
  
  <!-- Run sound_play -->
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>

</launch>
